langchain/callbacks/manager.py

```diff
@@ -66,7 +67,9 @@ def tracing_v2_enabled(
         "This is not yet stable and may change in the future."
     )
     cb = LangChainTracerV2()
-    session = cb.load_session(session_name)
+    if example_id:
+        cb.example_id = UUID(example_id)
```

Comment:
Wouldn't this be better as an optional constructor arg to the tracer?

Reply:
It would indeed, I believe